model/jpa/src/main/resources/META-INF/jpa-changelog-14.0.0.xml

```diff
@@ -38,30 +38,61 @@
 
     <changeSet author="keycloak" id="14.0.0-KEYCLOAK-18286">
```

Comment:
Do we need to keep this changeSet here?

Reply:
Not necessarily, but unless this is strictly needed, I prefer keeping this PR as is due to pipeline runs